backend/src/main/java/com/htecgroup/skynest/model/dto/CompanyDto.java

```diff
@@ -12,7 +13,10 @@ public class CompanyDto implements Serializable {
   private LocalDateTime createdOn;
   private LocalDateTime modifiedOn;
   private LocalDateTime deletedOn;
+  private String pib;
```

Comment:
is CompanyDto used anywhere?

Reply:
It isn't, It was created before so I just left it for possible future use. Should I just delete it for now?